rust-toolchain

```diff
@@ -0,0 +1 @@
+nightly-2022-07-29
```

Comment:
Seems unnecessary

Reply:
Did you set nightly as default? `#![feature(allocator_api)]` is unstable.